packages/components/psammead-podcast-promo/src/components/card-call-to-action.jsx

```diff
@@ -14,10 +14,10 @@ const CallToActionText = styled.span`
   color: ${C_METAL};
 `;
 const CardCallToAction = ({ children, ...props }) => (
-  <span aria-hidden="true">
-    <SeriesStackIcon />
+  <p>
+    <SeriesStackIcon focusable="false" />
```

Comment:
Curious what effect this has - it seems to be setting this value on a span?

I didn't think the span would be focusable anyway - the SVGs are on IE, but the SVG elements already have `focusable="false"`

Reply:
Ah so they do - https://github.com/bbc/psammead/blob/d1306b84060e19a30d58e7084b512da9a48bdc3b/packages%2Futilities%2Fpsammead-assets%2Fsrc%2Fsvgs%2FmediaIcons.jsx#L10 

Nice one I'll remove 👍